tests/lib/prepare-project.sh

```diff
@@ -68,17 +68,18 @@ fi
 
 # declare the "quiet" wrapper
 . "$TESTSLIB/quiet.sh"
+. "$TESTSLIB/dirs.sh"
```

Comment:
Give that this would not rely simplify what we have to put into spread.yaml unless we have a `environment-script` or `environment-include` element lets do the following:

 * We continue to use $TESTLIB/dirs.sh everywhere in task.yaml where needed for the moment
 * In parallel we start working on support for environment-script / environment-include for spread

@zyga Is this OK for you?